docs/github/getting-started.md

```diff
@@ -42,17 +42,15 @@ Read the official documentation on how to setup a
 
 Any subsequent steps assume you have read the above.
 
-#### Supported versions
-
-Unity Actions are based on the
-[unity3d](https://gitlab.com/gableroux/unity3d)
-images from
-[GabLeRoux](https://github.com/GabLeRoux).
-Any version in the
-[list](https://hub.docker.com/r/gableroux/unity3d/tags)
+#### Supported unity versions
+
+Unity Actions are using 
+[game-ci/docker](https://github.com/game-ci/docker/) 
+since `unity-builder` version 2. Any version in this 
+[list](https://hub.docker.com/r/unityci/editor/tags)
 can be used to test and build projects.
 
-It's generally considered good practice to use the same Unity version for Unity Actions as you do to develop your project.
+It's generally considered good practice to use the same Unity version for Unity Actions as you do to develop your project. No need to specify your unity version anywhere, `unity-builder` and `unity-test-runner` will use the one specified in `ProjectSettings/ProjectVersion.txt`.
```

Comment:
I would keep it rather short, and only document this explanation under unityVersion. 

The philosophy should always be trying to become zero-config, but that shouldn't mean more documentation.

Reply:
So we shouldn't even mention it's good practice, it does it by default now. I'll delete whole line. Good point 👍

Reply:
Even better!